cis/publisher.py

```diff
@@ -118,6 +124,42 @@ def _validate_profile_data(self):
         # Validate data prior to sending to CIS
         return validation.Operation(self.publisher, self.profile_data).is_valid()
 
+    def _reintegrate_profile_with_api(self):
+        person = api.Person(
+            person_api_config={
+                'audience': self.config('person_api_audience', namespace='cis'),
+                'client_id': self.config('oidc_client_id', namespace='cis'),
+                'client_secret': self.config('oidc_client_secret', namespace='cis'),
+                'oidc_domain': self.config('oidc_domain', namespace='cis'),
+                'person_api_url': self.config('person_api_url', namespace='cis')
+            }
+        )
+
+        # Retrieve the profile from the CIS API
+        vault_profile = person.get_userinfo(self.profile_data.get('user_id'))
+
+        if vault_profile is not None:
+
+            logger.debug('Vault profile retreived for existing vault user: {}'.format(vault_profile.get('user_id')))
+            publisher_groups = self.profile_data.get('groups')
+            vault_groups = vault_profile.get('groups')
+
+            reintegrated_groups = []
+
+            for group in publisher_groups:
```

Comment:
this only reintegrates groups - i understand that this is so that it can be generic, but for example mozillians.org has access to more fields than `user.groups`
say, what about tshirt size?

I'm blocking review on this one (though maybe I missed/misunderstood something) as it might break stuff

Reply:
mozillians.org is authoritative for that data so... no need to reintegrate.

Reply:
hah my bad i was confused